cookbooks/cdo-ruby/metadata.rb

```diff
@@ -1,9 +1,10 @@
-name             'cdo-ruby-2.0'
+name             'cdo-ruby'
 maintainer       'Code.org'
-maintainer_email 'geoffrey@code.org'
+maintainer_email 'will@code.org'
 license          'All rights reserved'
-description      'Installs/Configures cdo-ruby-2.0'
+description      'Installs/Upgrades Ruby'
```

Comment:
Could you make sure the user reading the description has a way to discover how to configure which version of ruby and bundler are installed?

If it's too long to document into the description, maybe you could just reference the detailed documentation in the description?